src/base/templates/base.hbs

```diff
@@ -281,11 +281,11 @@
     <script>
       (function(S, $){
         S.Config = {
-          defaultPlaceTypeName: '{{ default_place_type }}',
+          defaultPlaceTypeName: "{{ default_place_type }}",
           userToken: (
             S.bootstrapped.currentUser ?
-            'user:' + S.bootstrapped.currentUser.id :
-            "{{ user_token_json }}"),
+            "user:" + S.bootstrapped.currentUser.id :
+            "session:" + S.Util.cookies.get("sa-api-sessionid")),
```

Comment:
The point of this was just to restore what I believe to be feature parity pre-static site. If you look at anonymous comments submitted pre-static site, they store the session token value prefixed with `session:`. My understanding is this token could be used to identify all submissions in a single session, bit since the session token can change it can't be used to id users later.